packages/tools/devtools/devtools-example/webpack.config.js

```diff
@@ -8,12 +8,14 @@ const HtmlWebpackPlugin = require("html-webpack-plugin");
 const { merge } = require("webpack-merge");
 
 const sourcePath = path.join(__dirname, "src");
+const webpack = require("webpack");
 
 module.exports = (env) => {
 	const isProduction = env && env.production;
 
 	return merge(
 		{
+			mode: isProduction ? "production" : "development",
```

Comment:
I think we can always use "development" here. We never intend to actually publish this example.
```suggestion
			mode: "development",
```